docs/contributing/introduction-to-contributing.md

```diff
@@ -80,19 +81,31 @@ We actively welcome your pull requests. However, you must ensure that **you are
 8. If you improve the existing code, please let us know in your PR description.
 9. Completing the PR form is required. Make sure to fill in the PR title, description, [link to an issue](https://help.github.com/en/github/writing-on-github/autolinked-references-and-urls), and all the necessary areas.
 
-   - The title must begin with "feat:", "fix:", or anything related to your changes. <br /> **Tips**: You can follow your chosen option when [committing](#commits) your changes.
-
+   - The title must begin with `feat:`, `fix:`, or anything related to your changes. <br /> **TIP:** You can follow your chosen option when [committing](#commits) your changes.
    - Unsolicited code is welcomed, but an issue is required to announce your intentions.
 
-⚠️ **PRs will be marked as invalid and may be closed if:**  
+:::warning
+
+**PRs will be marked as invalid and may be closed if:**
+
+- the issue is not assigned to the contributor who opened the PR
+- no issue is linked to the PR
+- PR form is incomplete, or any section in the form is deleted
+- changes are made directly on the default (`main`, `beta`, etc.) branch
+
+:::
 
-- the issue is not assigned to the contributor who opened the PR  
-- no issue is linked to the PR  
+:::info
 
-> **IMPORTANT:** <br />
-> Please do not DM Maintainers or contact them outside of GitHub if you want a PR Review. If you haven't received a review within a week, please tag them in the PR comments to ask for an estimated time of review.
+**IMPORTANT** <br /> Please do not DM Maintainers or contact them outside of GitHub if you want a PR Review. If you haven't received a review within a week, please tag them in the PR comments to ask for an estimated time of review.
 
-> _**Note for maintainers**: All PRs need a label to assist automation. See [this guide](./triage-guide.md#adding-issue-and-pr-labels) to determine which labels to use._
+:::
+
+:::note
```

Comment:
I think the back to back "special" blocks might be too much. Let's leave the note as is, esp bc it's more for maintainers and not necessarily something contributors really need.